sp/src/game/server/hl2/env_headcrabcanister.cpp

```diff
@@ -33,6 +33,9 @@ ConVar sk_env_headcrabcanister_shake_radius( "sk_env_headcrabcanister_shake_radi
 ConVar sk_env_headcrabcanister_shake_radius_vehicle( "sk_env_headcrabcanister_shake_radius_vehicle", "2500" );
 
 #define ENV_HEADCRABCANISTER_TRAIL_TIME	3.0f
+#ifdef MAPBASE
+#define RANDOM_CRAB_TYPE 3
```

Comment:
What if a mod adds an additional headcrab type? If they do so after merging this change, they will need to know to leave an empty spot in the array at index 3. If they have already implemented it, they will need to update this value and then their FGD will need to be changed as well.

I would suggest making this number some arbitrarily high number so that there is no chance of collision. Or better yet, use -1 - since that's an invalid index there is no chance it could be used in the array.

Reply:
Fixed.